src/main/java/com/revature/services/UserServiceImpl.java

```diff
@@ -83,7 +83,21 @@ public User login(String name, String password) {
 		 }
 		
 	}
-
+	
+	public User validateUser(String username) {
+		String query = "SELECT * FROM USERS u where u.username='"+username+"'";
+		System.out.println("Query String : "+query);
+		Query q = em.createNativeQuery(query, User.class);
+		  System.out.println("Resultset  : "+q.getResultList());
+		  if((q.getResultList().size()) > 0) {
+		  User user = (User) q.getResultList().get(0);
+		  return user;
```

Comment:
validateUser() method definition to check whether the username already present in database